tests/unit/support/helpers/mount.ts

```diff
@@ -1,6 +1,7 @@
 import { mount as vtuMount } from '@vue/test-utils';
-import { Router, _RouteLocationBase } from 'vue-router';
+import type { Router, _RouteLocationBase } from 'vue-router';
 import { vi } from 'vitest';
+//@ts-ignore TODO: Remove this ignore when @nuxtjs/apollo is updated to support bundler
```

Comment:
Is this linked to an issue somewhere so we remember to do as much? Or can we do so?